lochness/rpms/__init__.py

```diff
@@ -293,9 +294,24 @@ def initialize_metadata(Lochness: 'Lochness object',
 
                 # Consent date
                 if rpms_consent_colname in df_measure.index:
-                    subject_dict['Consent'] = datetime.strptime(
+                    consent_tmp = datetime.strptime(
                             df_measure[rpms_consent_colname],
-                            '%d/%m/%Y %I:%M:%S %p').strftime('%Y-%m-%d')
+                            '%d/%m/%Y %I:%M:%S %p')
+
+                    if subject in consent_dict.keys():
+                        prev_consent = datetime.strptime(
+                            consent_dict[subject], '%Y-%m-%d')
+                        if prev_consent > consent_tmp:
```

Comment:
Shouldn't the condition be `<` ? @kcho

Reply:
`consent_tmp` is the consent date in a new line and `prev_consent` is the earliest consent date detected so far for the subject. Only when `consent_tmp` is smaller (earlier) than `prev_consent`, we will update consent_dict.